packages/lights.yaml

```diff
@@ -109,19 +107,16 @@ homeassistant:
     light.bedside_table_brightness:
       ignore_brightness: true
 
-    light.back_porch_fountain_lights:
-      notify_action: duck
-      fixed_brightness: 255
-    light.back_porch_fountain_blue:
-      fixed_brightness: 0
-    light.back_porch_fountain_brightness:
-      fixed_brightness: 0
+    light.back_porch_fountain_white:
+      ignore_brightness: true
     light.back_porch_fountain_green:
-      fixed_brightness: 0
+      ignore_brightness: true
```

Comment:
What does this `ignore_brightness` do?

Reply:
It opts lights out of [this automation](https://github.com/jnewland/ha-config/blob/6a0485694ec71a9995913f380e8d5465ca3b8e44/packages/lights.yaml#L368-L396), which sets lights' brightness to the value of an `input_number` after they are turned on.

Reply:
That's SOO Cool.  I didn't know you could make custom attributes using customize that way.  Really good piece of reference code.  Thanks!